src/js/components/Collapsible.js

```diff
@@ -0,0 +1,66 @@
+// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
+
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import TransitionGroup from 'react-addons-transition-group';
```

Comment:
No, it uses the same library (react transition group) as the `Animate` component, but this component has a different purpose. The Collapsible component is used to animate `height`. This is done through JavaScript. It's currently not possible to animate dynamic heights with CSS, e.g. `height: 0;` to `height: 100%;` does not work. I've also tried the `max-height` approach, but the animation/transition is very janky & not reliable depending on the panel content.